CocoaTypograf/Service/TypografService.swift

```diff
@@ -8,14 +8,26 @@
 
 import Foundation
 
+/// A protocol describing a typograf service.
 public protocol TypografService {
 
+    /// A handler for an processing a result of processed text or a failure.
```

Comment:
Some mess at the doc-comment — "A handler for an processing a result of processed text or a failure". Better to say "A closure for handling a result of a text processing whether the result might be a success or a failure".

Reply:
Fixed